src/OFS/History.py

```diff
@@ -56,7 +56,7 @@ def commit(self, object, transaction):
     def abort(*args, **kw):
         pass
 
-    tpc_begin = tpc_finish = abort
+    register = tpc_begin = tpc_finish = abort
```

Comment:
I think this can be left out: `register` was part of an old and no longer supported API, if I remember correctly.

Reply:
After reading the whole PR thread, I retract my comment. My memories seem to betray me. Time to get some fresh air.